src/main/frontend/src/features/CourseTopic.js

```diff
@@ -36,7 +36,7 @@ function CourseTopic (props) {
   }
 
   return (
-    <Row className="mt-4">
+    <Row className="mt-4 pb-5">
```

Comment:
Unfortunately, this adds padding to every row. This creates more to scroll so that's not ideal. We'll want to remove this.

Reply:
...the way I didn't even notice this 😩